backend/apps/admin/views.py

```
@@ -0,0 +1,88 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework.permissions import IsAdminUser
+from .services import ProductivityService
+from .serializers import ProductivitySerializer
+import csv
+from django.http import HttpResponse
+
+class ProductivityDashboardView(APIView):
+    permission_classes = [IsAdminUser]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)